types/tween.js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for tween.js v16.9.0
+// Type definitions for tween.js v17.2.0
```

Comment:
This appears to be a malformed header. Could you remove the line in the `tslint.json` file in this folder that says `dt-header: false` and fix any errors that come up? You will want this first line to say `// Type definitions for tween.js 17.2`. You are welcome to remove any other such line from the `tslint.json` file, though that might be a lot of work depending on what issues are being ignored. Thanks!

Reply:
Done